analyzers/tests/SonarAnalyzer.UnitTest/packages.lock.json

```diff
@@ -1,1829 +1,1780 @@
-        "dependencies": {
-          "System.Collections": "4.3.0",
-          "System.Diagnostics.Debug": "4.3.0",
-          "System.Diagnostics.Tools": "4.3.0",
-          "System.Globalization": "4.3.0",
-          "System.IO": "4.3.0",
-          "System.Reflection": "4.3.0",
-          "System.Resources.ResourceManager": "4.3.0",
-          "System.Runtime": "4.3.0",
-          "System.Runtime.Extensions": "4.3.0",
-          "System.Text.Encoding": "4.3.0",
-          "System.Threading": "4.3.0",
-          "System.Xml.ReaderWriter": "4.3.0"
-        }
-      },
-      "System.Xml.XmlDocument": {
-        "type": "Transitive",
-        "resolved": "4.3.0",
-        "contentHash": "lJ8AxvkX7GQxpC6GFCeBj8ThYVyQczx2+f/cWHJU8tjS7YfI6Cv6bon70jVEgs2CiFbmmM8b9j1oZVx0dSI2Ww==",
-        "dependencies": {
-          "System.Collections": "4.3.0",
-          "System.Diagnostics.Debug": "4.3.0",
-          "System.Globalization": "4.3.0",
-          "System.IO": "4.3.0",
-          "System.Resources.ResourceManager": "4.3.0",
-          "System.Runtime": "4.3.0",
-          "System.Runtime.Extensions": "4.3.0",
-          "System.Text.Encoding": "4.3.0",
-          "System.Threading": "4.3.0",
-          "System.Xml.ReaderWriter": "4.3.0"
-        }
-      },
-      "SonarAnalyzer": {
-        "type": "Project",
-        "dependencies": {
-          "Google.Protobuf": "3.6.1",
-          "Google.Protobuf.Tools": "3.6.1",
-          "Microsoft.CodeAnalysis.Workspaces.Common": "1.3.2",
-          "Microsoft.Composition": "1.0.27",
-          "System.Collections.Immutable": "1.1.37"
-        }
-      },
-      "sonaranalyzer.cfg": {
-        "type": "Project",
-        "dependencies": {
-          "Microsoft.CodeAnalysis.CSharp.Workspaces": "1.3.2",
-          "Microsoft.Composition": "1.0.27",
-          "System.Collections.Immutable": "1.1.37"
-        }
-      },
-      "sonaranalyzer.csharp": {
-        "type": "Project",
-        "dependencies": {
-          "Microsoft.CodeAnalysis.CSharp.Workspaces": "1.3.2",
-          "SonarAnalyzer": "1.0.0",
-          "SonarAnalyzer.CFG": "1.0.0",
-          "System.Collections.Immutable": "1.1.37"
-        }
-      },
-      "sonaranalyzer.ruledescriptorgenerator": {
-        "type": "Project",
-        "dependencies": {
-          "SonarAnalyzer": "1.0.0",
-          "SonarAnalyzer.Utilities": "1.0.0",
-          "System.Collections.Immutable": "1.1.37"
-        }
-      },
-      "sonaranalyzer.utilities": {
-        "type": "Project",
-        "dependencies": {
-          "SonarAnalyzer": "1.0.0",
-          "SonarAnalyzer.CSharp": "1.0.0",
-          "SonarAnalyzer.VisualBasic": "1.0.0"
-        }
-      },
-      "sonaranalyzer.visualbasic": {
-        "type": "Project",
-        "dependencies": {
-          "Microsoft.CodeAnalysis.VisualBasic.Workspaces": "1.3.2",
-          "SonarAnalyzer": "1.0.0",
-          "System.Collections.Immutable": "1.1.37"
-        }
-      }
-    }
-  }
+{
```

Comment:
there's something weird about the changes is this lockfile - with whitespace changes it's 3.6K LOC diff, ignoring whitespace changes it's 2K LOC diff. 

Is it because of a GH diff bug or because the whitespace characters changed when regenerating it?

Reply:
(it makes reviewing the diff difficult, although I can just filter the whitespace here in GH; but if it's easy to revert the unnecessary changes , it would be nice to do it)

Reply:
I even tried to staged only changed parts of this file, it still made a full change. Probably it is generating different line endings?

Reply:
Re-checked my git settings, resetted the file to master and did regeneration one more time - no effect. In the IDE I see only particular changes, but on github - still the whole file